app/config/settings.py

```diff
@@ -289,6 +289,10 @@
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 SECURE_SSL_REDIRECT = True
 
+# Set permissive cross-origin header to allow session control of windows opened that
+# go to a different subdomain (workstations)
+SECURE_CROSS_ORIGIN_OPENER_POLICY = None
+
```

Comment:
We probably want to control this per domain, so better take it from the environment variables.

```suggestion
SECURE_CROSS_ORIGIN_OPENER_POLICY = os.environ.get("SECURE_CROSS_ORIGIN_OPENER_POLICY", "same-origin")
```